src/ui/util/TableConfig.ts

```diff
@@ -0,0 +1,103 @@
+import { idb } from "../../worker/db";
+import getCols, { MetaCol } from "./columns/getCols";
+import { cloneDeep, uniq } from "lodash-es";
+import { g } from "../../worker/util";
+
+export class TableConfig {
+	get ratingsNeeded(): string[] {
```

Comment:
This isn't Java, just let people access the variables directly, getters wrapping them are not needed :)

Reply:
This is java now! Actually the reason I did this was because I didn't want them to be externally settable, ensuring it always has the correct values needed for the current columns. However if it was declared as: `readonly ratingsNeeded` then it would be settable anywhere but the constructor.

This all doesn't matter thou as this isn't a distributed package, so I just removed the getters.